packages/ag-charts-community/src/chart/chart.ts

```diff
@@ -258,18 +261,19 @@ export abstract class Chart extends Observable {
         const container = resources?.container ?? options.processedOptions.container ?? undefined;
 
         const root = new Group({ name: 'root' });
-        const titleGroup = new Layer({ name: 'titles', zIndex: ZIndexMap.SERIES_LABEL });
         // Prevent the scene from rendering chart components in an invalid state
         // (before first layout is performed).
         root.visible = false;
-        root.append(titleGroup);
+        root.append(this.backgroundRoot);
```

Comment:
So I don't think these changes are necessarily needed after #2705, we just need to add a z-index on the series root.

Reply:
Updated